__tests__/app.test.js

```diff
@@ -71,3 +70,19 @@ describe('/api/articles/:article_id', () => {
   });
 });
 
+describe('/api/articles', () => {
+  test('GET:200 gets all articles with specified columns', () => {
+    return request(app)
+      .get('/api/articles/')
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.articles.length).toBe(13);
```

Comment:
Well done for asserting against the length here to ensure the test does not pass by default if we get an empty array back! :smile: